src/actions/article.js

```diff
@@ -9,10 +7,16 @@ import { QUERY_FETCH_BUCKET_ITEM } from '../constants/queries';
  */
 export const viewArticle = (content) => { return { type: VIEW_ARTICLE, content }; };
 
-export const getArticle = (bucketId, itemId) => {
+export const getArticle = (userId, bucketId, itemId) => {
   return (dispatch) => {
-    return graphqlService.query(QUERY_FETCH_BUCKET_ITEM, {'id': bucketId, 'itemId': itemId, 'itemType': 'article'})
-      .then((data) => { console.log('DATA :', data); dispatch(viewArticle(data.data.viewer.searchItem.tile)); })
-      .catch((err) => console.log(err));
+    return new Promise((resolve, reject) => {
+      // JQuery imported at index.html and intex.template.html
+      $.getJSON(
+        // `https://numo-search-results.s3.amazonaws.com/ci/${userId}/${bucketId}/${itemId}.json`,
+        'https://s3-eu-west-1.amazonaws.com/numo-search-results/ci/TESTUSERID/testhotel/article.json',
```

Comment:
@Kumjami this is the hard-coded url you were mentioning.
Is the _plan_ to fetch the articles from S3 directly or to go through GraphQL?